moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js

```diff
@@ -70,7 +71,10 @@ const SignalProjectTable = () => {
     const signal_ids = [];
     if (project?.moped_proj_features.length) {
       project.moped_proj_features.forEach(feature => {
-        signal_ids.push(feature?.location?.properties?.signal_id);
+        let signal = feature?.location?.properties?.signal_id;
```

Comment:
Unimportant, but it would be nice to stick to `const` conventions. Reading this code hints at me that signal is going to change later, even if it doesn't and even if it is never going to be a problem.